### ISSUE TITLE
Forms are not attaching files to ticket

### ISSUE THREAD

**Issue by wiictorhub** [pluginsGLPI/formcreator#3211]
Hi!

After migrating GLPI from version 9.5.4 to 10.0.6, the forms stopped attaching files the moment they generate the ticket, emphasizing that the file is uploaded normally.

In addition, in the Items tab of the ticket that was opened by the form, the error that is in the image and also described below is occurring.

PS: in the Items tab of other tickets with common assets (computers, printers, etc) the error reported in PHP is not occurring, only when it is generated by the form and the attached item is "Form Response"

PHP ERRORS:

PHP Warning (2): Undefined array key "states_id" in /var/www/html/glpi/src/Item_Ticket.php at line 509

PHP Warning (2): Undefined array key "locations_id" in /var/www/html/glpi/src/Item_Ticket.php at line 511

My system settings:

Ubuntu Server 20.04
GLPI v10.0.6
PHP 8.1
Formcreator 2.13.4
![Erro PHP](https://user-images.githubusercontent.com/35465037/226040084-dcd1995d-1516-4e37-adb5-d35d80b0ad5a.JPG)

**Comment by btry** [comment]
Hi

Can you try the latest revision of the plugin ? You need to build it (composer, yarn). If you don't know how to do it, let me know and I'll send you a build.

Remember that it shall be used on a testing instance.

**Comment by wiictorhub** [comment]
Hello

I installed the latest version from the site. If it's another revision that needs to be compiled, I would kindly appreciate your help.

Thanks!

**Comment by wiictorhub** [comment]
> Hi
> 
> Can you try the latest revision of the plugin ? You need to build it (composer, yarn). If you don't know how to do it, let me know and I'll send you a build.
> 
> Remember that it shall be used on a testing instance.

Hello

I installed the latest version from the site. If it's another revision that needs to be compiled, I would kindly appreciate your help.

Thanks!

**Comment by btry** [comment]
Hi, 

here is a build of the latest revision. Let me know if the issue persists with it.

[glpi_formcreator_2.13.5-dev.zip](https://github.com/pluginsGLPI/formcreator/files/11015325/glpi_formcreator_2.13.5-dev.zip)

**Comment by wiictorhub** [comment]
> Hi,
> 
> here is a build of the latest revision. Let me know if the issue persists with it.
> 
> [glpi_formcreator_2.13.5-dev.zip](https://github.com/pluginsGLPI/formcreator/files/11015325/glpi_formcreator_2.13.5-dev.zip)

Hi!

I just discovered the problem! I usually customize the body of the tickets in the Targets of the form and now it looks like I need to put the ##answer## tag from the upload response in the body of the Target description. As I didn't need to do that, I didn't notice that this would be the problem and I only found out because I created a form from scratch leaving the ##FULLFORM##.

Thanks for your help!
